.gitignore

```diff
@@ -0,0 +1,3 @@
+cp.sh
+PHPMailer-master/
+mailservice.php
```

Comment:
Why do we need these?

Reply:
We don't need them, that's why they are in .gitignore. I used it to facilitate my git usage. Probably shouldn't be here, but for our purposes I think that's OK :D